reservation/views.py

```diff
@@ -34,3 +37,25 @@ def book_a_ticket(request, pk, hall_pk):
 
     return render(request, 'reservation/reserve.html', {'form': form,})
 
+def get_csv_file(request, pk):
```

Comment:
Currently, the view is accessible by anyone; if I know the id of an user, i can get a csv with his reservations. Allow only logged in users to get their reservations without providing the id as a parameter. Get the logged in user automatically, no parameters.